sdk/core/switcher/entry.S

```diff
@@ -26,6 +26,8 @@
 #define SPILL_SLOT_pcc 24
 #define SPILL_SLOT_SIZE 32
 
+switcher_code_start:
```

Comment:
This is still assuming that `entry.S` is the first `.o` to place anything in the `.text` section, isn't it?  Can we not use `__compartment_code_start` from the linker script?

Reply:
No, this .o file assembles to the equivalent of a .compartment file.  It does not have the first-stage linking that a normal compartment has.  Changing that would require a separate linker script because the firmware linker script needs to find some symbols in this file.